Gateway/Http/Client/RefundClient.php

```diff
@@ -103,9 +103,9 @@ public function placeRequest(TransferInterface $transferObject): array
     private function refund(array $payloadData, bool $isFullRefund): Payment
     {
         if ($isFullRefund) {
-            $refund = $this->alma->getDefaultClient()->payments->fullRefund($payloadData['payment_id'], $payloadData['merchant_id'], 'Full refund with Magento 2 module');
+            $refund = $this->alma->getDefaultClient($payloadData['store_id'])->payments->fullRefund($payloadData['payment_id'], $payloadData['merchant_id'], 'Full refund with Magento 2 module');
```

Comment:
<img width="933" alt="image" src="https://user-images.githubusercontent.com/97046219/181629502-c4349775-ed1b-44b0-8b0f-24e78f1b183f.png">

Are you sure , payloadData['store_id'] can be null but it can't be undefined ?

Reply:
Done fore Refund